pkg/kubelet/cm/dra/manager.go

```diff
@@ -146,30 +134,26 @@ func (m *ManagerImpl) prepareContainerResources(pod *v1.Pod, container *v1.Conta
 	return nil
 }
 
-// Allocate calls plugin NodePrepareResource from the registered DRA resource plugins.
-func (m *ManagerImpl) Allocate(pod *v1.Pod, container *v1.Container) error {
-	// The pod is during the admission phase. We need to save the pod to avoid it
-	// being cleaned before the admission ended
-	m.setPodPendingAdmission(pod)
-
+// PrepareResources calls plugin NodePrepareResource from the registered DRA resource plugins.
+func (m *ManagerImpl) PrepareResources(pod *v1.Pod, container *v1.Container) (*DynamicResourceContainerOptions, error) {
 	// Prepare resources for init containers first as we know the caller always loops
 	// through init containers before looping through app containers. Should the caller
 	// ever change those semantics, this logic will need to be amended.
```

Comment:
We don't need this comment as there is no notion of "reused devices" with DRA.

Reply:
Thank you for the review! Removed